app/core/items.ts

```diff
@@ -216,5 +220,91 @@ export const ItemEffects: { [i in Item]?: Effect[] } = {
       pokemon.addAttackSpeed(-5 * stacks, pokemon, 0, false)
       pokemon.count.defensiveRibbonCount = 0
     })
+  ],
+
+  [Item.COMFEY]: [
+    new OnItemGainedEffect((pokemon) => {
+      const simulation = pokemon.simulation
+      const comfey = simulation.board.find(
+        (x, y, e) => e.passive === Passive.COMFEY && e.team === pokemon.team
+      )
+      if(!comfey){
+        pokemon.items.delete(Item.COMFEY)
+        return // can't steal comfey
```

Comment:
This is a major change, if we agree to do it at least it needs to be documented in patch notes and PR description

Reply:
I meant to ask about this. I guess I hallucinated that I did. Will discuss on Discord.